azerothcore/azerothcore-wotlk
# fix:(DB/world) Re-arrange location of Webwinder Path's signpost gps so they sit on the sign

Fixing the webwinder path signpost gps locations so they actually sit on the post rather than floating



## Changes Proposed:
This PR proposes changes to:

If your pull request promotes complex changes that require a detailed explanation, please describe them in detail specifying what your solution is and what is it meant to address.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11992

## SOURCE:

The changes have been validated through:

## Tests Performed:

This PR has been:


## How to Test the Changes:



1. .tele webwinder
2. Walk a bit south and check the signpost, make sure the three pieces nailed on to the sign aren't floating in space and appear attached to the post.


## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
